src/lib/components/Toggle/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file is part of React-SearchKit.
+ * Copyright (C) 2020 CERN.
+ *
+ * React-SearchKit is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import { connect } from '../../store';
+import { updateQueryFilters } from '../../state/actions';
+import ToggleComponent from './ToggleElement';
+
+const mapDispatchToProps = dispatch => ({
+  updateQueryFilters: filter => dispatch(updateQueryFilters(filter)),
+});
+
+export const Toggle = connect(
+  state => ({
+    userSelectionFilters: state.query.filters
```

Comment:
Not used :)

Reply:
Not sure what you mean, but it is used right?

Reply:
Yes it was updated I think...Eitherway, I will resolve it!